### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by KirtiVardhan2019**
Production- Assignment2

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Extracted data, pre processing data, running a regression pipeline, calculate error, choose and explain mode. 

## What did you learn from the changes you have made?
Building pipelines,model selecting and explanation.

## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by fyellow — APPROVED** [review]
Good Job but there are also many “unknown category” warnings from your OneHotEncoder — these come from categories in the test set that weren’t in the training data; try using handle_unknown='ignore' (which you already did, but double-check your categorical columns).  Assignment approved
